pyscript.core/tests/integration/test_01_basic.py

```diff
@@ -151,20 +159,24 @@ def test_execution_in_order(self):
             "four",
         ]
 
+    @skip_worker("NEXT: something very weird happens here")
     def test_escaping_of_angle_brackets(self):
         """
         Check that script tags escape angle brackets
         """
         self.pyscript_run(
             """
             <script type="py">import js; js.console.log("A", 1<2, 1>2)</script>
-            <script type="py">js.console.log("B <div></div>")</script>
+            <script type="py">import js; js.console.log("B <div></div>")</script>
```

Comment:
the only weird thing I see is that we use the worker global `console` while *maybe* we check/test the main thread one ... in that case it's `from pyscript import window; window.console.log(...)`